### DISCUSSION TITLE
How to display bar chart background like display in following image

### DISCUSSION THREAD

**Discussion by AbhishekMuknak** [Abhinandan-Kushwaha/react-native-gifted-charts#904] [Q&A] [answered]
![IMG_20241112_104756068](https://github.com/user-attachments/assets/2b9968e0-12b8-4d36-8910-ab826edf2655)
I want show gray bar 100 percent if there is no data for given value and if value is present then fill the bar with given value in green and remaining in gray color

**Comment by ak-spotter** [comment, marked as answer]
Hi @AbhishekMuknak This can be achieved using the `showVerticalLines` and `verticalLinesThickness` props.
If the vertical lines are not aligned with the bars, you can use the `verticalLinesShift` prop. The color of vertical lines is gray by default which can be changed using the `verticalLinesColor` prop.

Here's an example-

```js
<BarChart
  data={[{ value: 5 }, { value: 3 }, {value:8}]}
  barWidth={15}
  frontColor={'navy'}
  roundedTop
  showVerticalLines
  verticalLinesThickness={15}
  verticalLinesStrokeLinecap="round"
/>
```
The output of above code is-
 
<img width="192" alt="Screenshot 2024-11-12 at 2 12 34 PM" src="https://github.com/user-attachments/assets/5d114959-b58b-4905-a8ae-62f8afacacb2">
